toolchain/sem_ir/function.h

```diff
@@ -44,6 +44,8 @@ struct FunctionFields {
   // this function.
   VirtualModifier virtual_modifier;
 
+  uint16_t virtual_index = 0;
```

Comment:
Comment?

Also, is 16-bits enough? I would have defaulted to 32-bits and maybe initialize to `-1` to avoid any confusion that `0` is meaningful for non-virtual functions?

Reply:
oh, yeah, sorry about the missing comment, added.

Re: 16-bit, was trying to fit it in the existing padding, but the structure's probably large enough that's not a major concern, so bumped it up to 32 and added the -1 sentinel to distinguish unintended usage